lib/Elastica/Aggregation/ExtendedStatsBucket.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Elastica\Aggregation;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Class ExtendedStatsBucket.
```

Comment:
That does not add more info about the class :)
Can you add a more descriptive text here?